data/settings_mapping.json

```diff
@@ -206,7 +206,8 @@
             "shuffle_expensive_merchants",
             "shuffle_frog_song_rupees",
             "shuffle_loach_reward",
-            "shuffle_individual_ocarina_notes"
+            "shuffle_individual_ocarina_notes",
+            "max_upgrade_items"
```

Comment:
This really isn't a shuffle setting. My first instinct is Other → Misc, second is Other → Timesavers, and third is near the top of Detailed Logic.

Reply:
I guess Other → Misc makes more sense, I'll move it there